src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+//an interface that allows user to access data on a read-only, immutable basis
```

Comment:
Good that you write the header comment, but try to follow the coding standard for this. Check out ReadOnlyPerson interface for reference.